### DISCUSSION TITLE
Status not changing

### DISCUSSION THREAD

**Discussion by JoshuaBarawa** [devlikeapro/waha#183] [Q&A]
Hi
After updating to the latest WAHA version my session status does not change to WORKING after scanning the QR code successfully.
Even though I can get my WhatsApp screenshot successfully, I cannot send a message. When I try to send a message I get this error:
{
  "statusCode": 500,
  "timestamp": "2023-08-23T09:18:52.929Z",
  "exception": {
    "stack": "Error: Evaluation failed: TypeError: Cannot read properties of undefined (reading 'WidFactory')\n    at __puppeteer_evaluation_script__:2:42\n    at ExecutionContext._evaluateInternal (/app/node_modules/whatsapp-web.js/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:221:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async ExecutionContext.evaluate (/app/node_modules/whatsapp-web.js/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:110:16)\n    at async Client.sendMessage (/app/node_modules/whatsapp-web.js/src/Client.js:869:28)\n    at async /app/node_modules/@nestjs/core/router/router-execution-context.js:46:28\n    at async /app/node_modules/@nestjs/core/router/router-proxy.js:9:17",
    "message": "Evaluation failed: TypeError: Cannot read properties of undefined (reading 'WidFactory')\n    at __puppeteer_evaluation_script__:2:42",
    "name": "Error"
  },
  "request": {
    "path": "/api/sendText",
    "method": "POST",
    "body": {
      "chatId": "254717916656@c.us",
      "mentions": [
        "123456789@c.us"
      ],
      "text": "Hi there!",
      "session": "joshua"
    },
    "query": {}
  },
  "version": {
    "version": "2023.8.21",
    "tier": "PLUS",
    "browser": "/usr/bin/chromium"
  }
}

Please help

**Comment by allburov** [comment]
Hi! Working on that 
https://github.com/devlikeapro/whatsapp-http-api/issues/182

**Reply by JoshuaBarawa** [reply]
Please update me on any updates regarding the situation.
Thanks

**Reply by allburov** [reply]
Hi! There's new `2023.8.23` with the fix available, try this one and let us know if it helped!

**Reply by JoshuaBarawa** [reply]
Should I pull the docker image again with the same credentials I used?

**Reply by allburov** [reply]
Yes, use the latest credentials from Patreon on Boosty. I think for you it's the same, you have a pretty recent version

**Reply by JoshuaBarawa** [reply]
okay sure
